<launch>
	<node pkg="sound_play" type="soundplay_node.py" name="sound_play" />
	<node pkg="wm_voice_generator" type="wm_voice_component_short.py" name="wm_voice_generator" />
	<node pkg="reha_game" type="game2.py" name="reha_game" args="--width 320 --height 280 --color yellow" output="screen" />
</launch>
